pkg/manager/manager.go

```diff
@@ -388,3 +402,26 @@ func (m *manager) DetachRepresentor(conf *types.PluginConf) error {
 	log.Info().Msgf("Detaching rep %s from the bridge %s", conf.Representor, conf.Bridge)
 	return m.nLink.LinkSetNoMaster(rep)
 }
+
+// configure port VLAN id for link
+func bridgePVIDVlanAdd(nlink NetlinkManager, link netlink.Link, vlanID int) error {
```

Comment:
can we move the function below to utility module?

Reply:
I can't check right now, but I think that we will have cyclic import if we do this.  Utilits package will need to import manager package to have access to NetlinkManager interface.

Reply:
I checked, if we simply move these functions to the utils package we will have an import cycle.  We can move all "wrappers" (MyNetlink, MyLittleSriov, pciUtils) and helpers for them from "manager" package to somewhere else, but these changes are probably out of the scope of this PR. 
I moved "bridge**" helper functions to be near MyNetlink wrapper, I think this helps to find them easier and understand that they are used together.